extensions/dacpac/src/wizard/common/fileLocationHelper.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as constants from '../common/constants';
+
+/**
```

Comment:
How man of these are getting used multiple times/have multiple callers?  Would it be cleaner/easier to follow to have one or two multi-line methods instead of five one-liners?

Reply:
I can clean it up. This was mostly copied from what was added in sql database projects